part2.asm

```diff
@@ -0,0 +1,58 @@
+# COMP3410 Program Template
+# Author: Aaron Marshall
+# Assignment: PA[1] part 2
+# Date: 2/10/14
+# takes two inputted number and adds, subs, and mults
+	.data
+input: .asciiz "Input two integers.\n"
+subtext: .asciiz "\nSubtracting......\n"
+addtext: .asciiz "\nAdding...........\n"
+multitext: .asciiz "\nMultiplying......\n"
+goodbye: .asciiz "\nBye..............\n"
+
+	.text
+  la $a0, input  #output of input text
+  li $v0, 4
+  syscall
+  
+  li $v0, 5 #it wants a number
+  syscall
+  add $s0, $zero, $v0 #adds number to register
+  
+  li $v0, 5 #get the second number
+  syscall
+  add $s1, $zero, $v0 #adds sec. to register
+  
+  la $a0, addtext #prints the addtext
+  li $v0, 4
+  syscall
+  
+  add $s2, $s0, $s1 # does the math then prints the result
+  add $a0, $s2, $zero
```

Comment:
-8 Instructions require that your calculations be stored in memory and retrieved from memory for printing.